lerna.json

```diff
@@ -1,6 +1,6 @@
 {
   "packages": ["build/packages/*"],
-  "version": "14.4.2",
+  "version": "999999.0.3",
```

Comment:
Is this the correct version?

Reply:
No, that's a mistake. I didn't realize local publish updated this file. Reverting...